extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/SparkRangerAdminPlugin.scala

```diff
@@ -131,4 +140,39 @@ object SparkRangerAdminPlugin extends RangerBasePlugin("spark", "sparkSql")
       }
     }
   }
+
+  def getOrCreateRangerPlugin(catalog: Option[String] = None): RangerBasePlugin = synchronized {
```

Comment:
getOrCreate

Reply:
Alright. I put RangerPlugin as suffix here is to distinguish `SparkRangerAdminPlugin` itself and `RangerBasePlugin`.

Reply:
But in usages, it will become `getOrCreate().isAccessAllowed(request)` which is misleading in some way.